settings.py

```diff
@@ -113,9 +113,11 @@ def debug_toolbar_enabled(request):
     'client_id': 'cd5a21fafacc4744',
     'client_secret':
         '4db6f78940c6653d5b0d2adced8caf6c6fd8fd4f2a3a448da927a54daba7d401',
+    'content_host': 'https://stable.dev.lcip.org',
+    'login_uri': 'http://olympia.dev/api/v3/accounts/login/',
+    'oauth_host': 'https://oauth-stable.dev.lcip.org/v1',
+    'profile_host': 'https://stable.dev.lcip.org/profile/v1',
```

Comment:
The FxA client wants `contentHost`, etc so I made those `_host`. The host ones are really prefixes for an API not a usable URL. The `_uri` ones are specific URLs on AMO.